src/App.js

```diff
@@ -77,11 +87,81 @@ function App() {
         }).then(() => refreshPokemonCards());
     }
 
+    function addToCollection(card) {
+        let cardMatch = false;
+        collectedArray.forEach((item) => {
+            if (item.collectedCard.id === card.id) {
+                let collectionObj = {
+                    user: item.user,
+                    quantity: item.quantity + 1,
+                    collectedCard: card.id,
+                };
+                fetch(
```

Comment:
this can be a future ticket, or you can do it now. 
But you may want to make a helper JS file with all your fetch functions in it to remove a lot of that from this file
So essentially make a file with a bunch of exported functions called like getPokemonCards(), getCollections() etc and then import them into this file so that you can just call them by function name